avocado/utils/multipath.py

```diff
@@ -71,7 +71,9 @@ def device_exists(mpath):
     """
     Checks if a given mpath exists.
 
+    :param mpath: The multipath path
     :return: True if path exists, False if does not exist.
+    :rtype: Boolean
```

Comment:
I am not sure what is the de facto standard for avocado docstrings, at least for our plugin we use `bool`, `str`, `int` and so on. Some of the docstrings below seem to be missing `rtype` for return types, as well as type descriptions. But this might be squeezing your contribution too much, just noting down here because the current docstring handling in this module is not uniform.